### PR TITLE
Json dieases

### PR DESCRIPTION
**Pull request by dolevfarhi**
Json dieases branch - 
Includes modification in the data mentioned in JSON file

### CODE REVIEW

File: Ingredients.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"UK" :{
```
Comment (atrope) [inline Ingredients.json]: Remove This file. no need for it.

File: onepage/js/dis.json [hunk 2]
```diff
@@ -4,18 +4,14 @@
     "ingredients": [
       {
         "name": "oil",
```
Comment (atrope) [inline onepage/js/dis.json:6]: missing good things for all diseases.

File: onepage/js/dis.json [hunk 3]
```diff
@@ -91,26 +59,11 @@
     "name": "tuberculosis",
     "ingredients": [
       {
-        "name": "chicken",
+        "name": "vitamin D",
```
Comment (atrope) [inline onepage/js/dis.json]: vitamin D is  not an ingredient

**Review by atrope — CHANGES_REQUESTED** [review]

File: Ingredients.json (hunk 1 above)
Comment (dolevfarhi) [reply Ingredients.json]: done

File: onepage/js/dis.json (hunk 2 above)
Comment (dolevfarhi) [reply onepage/js/dis.json:6]: done

File: onepage/js/dis.json (hunk 3 above)
Comment (dolevfarhi) [reply onepage/js/dis.json]: done

**Review by atrope — APPROVED** [review]

**Pull request merged**